### PR TITLE
Adding patient basis field to base configuration

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-5536](https://jira.cms.gov/browse/MAT-5536)
(Optional) Related Tickets:

### Summary

image

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by sb-cecilialiu — DISMISSED** [review]

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx [hunk 1]
```diff
@@ -277,28 +275,26 @@ describe("Base Configuration component", () => {
...
 
     expect(screen.getByText("Structure")).toBeInTheDocument();
     const target = screen.getByText("Structure");
 
-    act(() => {
-      userEvent.click(target);
-    });
+    userEvent.click(target);
+
+    // setting patient basis to true
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx]: I think this should be false.

**Review by adongare — DISMISSED** [review]

File: src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx (hunk 1 above)
Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx]: Story specifies it to be true by default

Comment (adongare) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx]: but it seems the following statement set it to false. Probably comment and the click event is a mismatch?

Comment (RohitKandimalla) [reply src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.test.tsx]: ooh, this is the test case, my bad cool. I will update it.

**Review by adongare — APPROVED** [review]

**Pull request merged**